<ui version="4.0" >
 <class>LaueControls</class>
 <widget class="QWidget" name="LaueControls" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Laue</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title" >
    <string>Laue Simulation</string>
   </property>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>160</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Detection Level</string>
    </property>
   </widget>
   <widget class="QSlider" name="DetLevelSlider" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>180</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>100</number>
    </property>
    <property name="value" >
     <number>0</number>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="DetLevelSpinBox" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>180</y>
      <width>63</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="SatLevelSlider" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>230</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>100</number>
    </property>
    <property name="value" >
     <number>100</number>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>210</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Sat. Level</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ShowSimulationCheckbox" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>260</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Show simulation</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>110</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Simulated spot size</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="SpotSizeSpinBox" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>130</y>
      <width>63</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Sample film distance (mm)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="SatLevelSpinBox" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>230</y>
      <width>63</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>100</number>
    </property>
    <property name="value" >
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Calculation Accuracy</string>
    </property>
   </widget>
   <widget class="QSlider" name="SpotSizeSlider" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>130</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>100</number>
    </property>
    <property name="value" >
     <number>0</number>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="AccuracySpinBox" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>80</y>
      <width>63</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>10</number>
    </property>
   </widget>
   <widget class="QSlider" name="AccuracySlider" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>80</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>10</number>
    </property>
    <property name="pageStep" >
     <number>1</number>
    </property>
    <property name="value" >
     <number>0</number>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sampleFilmD" >
    <property name="geometry" >
     <rect>
      <x>200</x>
      <y>30</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="singleStep" >
     <double>0.050000000000000</double>
    </property>
    <property name="value" >
     <double>35.000000000000000</double>
    </property>
   </widget>
   <widget class="QCheckBox" name="ShowSimulationIntensities" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>290</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Plot intensities</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ShowCharacteristicCheckBox" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>320</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Show Characteristic Intensities</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="IntensityFilmLikeCheckBox" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>350</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Show intensities as color scale</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
